cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -2065,6 +2090,20 @@ private List<Field<?>> getProgramRunInvertedTimeKey(String recordType, ProgramRu
     return fields;
   }
 
+  private List<Field<?>> getProgramRunInvertedTimeKeyWithNoVersion(String recordType,
```

Comment:
This method name is too long. Besides, why not just have a parameter to indicate if version should be included or not?

Reply:
good idea, will update

Reply:
Agree. Just use overloading, it speaks by itself